### PR TITLE
Weather envelope

### PR DESCRIPTION
**Pull request by AkaMrmuffin**
Add a built-in weather envelope for the weather checks. The specific thresholds for the weather envelopes need to be discussed.

### CODE REVIEW

**Comment by AkaMrmuffin** [comment]
#21

File: LDAR_Sim/model_code/weather/weather_lookup.py [hunk 1]
```diff
@@ -71,10 +71,47 @@ def deployment_days(self, method_name, start_date, start_work_hour=0, consider_w
         DD = deployment day
         """
 
+        # build-in Weather envelopes
+        temp_env = [0, 0]
```
Comment (KeeganShaw-GIS) [inline LDAR_Sim/model_code/weather/weather_lookup.py:75]: Should be set in Default Parameters , and Temperature low should be less than 0

**Review by KeeganShaw-GIS — CHANGES_REQUESTED** [review]
Nice work, update all Default Parameters to include default ranges, and update the documentation, then we can merge.

**Comment by KeeganShaw-GIS** [comment]
Added into followup PR #111

**Pull request closed without merging**